legacy/src/app/partials/nodedetails/deploy-options.html

```diff
@@ -26,6 +26,9 @@
             Read more
           </a>
         </label>
+        <p class="p-form-help-text u-no-margin--top" style="padding-left: 2rem">
```

Comment:
Probably worth an issue to Vanilla to align help text with checkbox labels

Reply:
Done! https://github.com/canonical-web-and-design/vanilla-framework/issues/3402